app/models/upload.rb

```diff
@@ -544,6 +509,25 @@ def self.migrate_to_new_scheme(limit: nil)
     problems
   end
 
+  def self.extract_upload_ids(raw)
```

Comment:
Doesn't look like this method is tested? I know it is indirectly because it's called in Draft, ReviewableQueuedPost and others, but having a test with a raw containing 1 URL matching each `InlineUploads.url_regexps` would ensure less regressions.